src/models/elan1-r1/README.md

```diff
@@ -0,0 +1,29 @@
+# Eland 1U (elan1-r1)
+
+![Eland 1U](./img/elan1-r1-frontports.avif)
+
+The System76 Eland 1U is a rack-mount server with the following specifications:
+
+- Operating System
+    - Ubuntu Server 22.04 LTS (64-bit)
+- CPU options
+    - Supports AMD EPYC™ 7003 series processor family
+    - Single processor
+- Memory
+    - Up to 4096GB (16x256GB) ECC DDR4 @ 3200 MHz
+- Networking
+    - 2x 1GbE LAN ports (1 x Intel® I350-AM2)
+    - 1x 10/100/1000 management LAN
+- Expansion
+    - 1 x PCIe x16 slot (Gen4)
+- Ports
+    - Front: 2 x USB 3.0, 1 x Power button with LED, 1 x ID button with LED, 1 x NMI button, 1 x Reset button, 2 x LAN activity LEDs, 1 x HDD activity LED, 1 x System status LED
+    - Rear: 3 x USB 3.0, 1 x VGA, 1 x COM, 2 x RJ45, 1 x MLAN, 1 x ID button with LED
```

Comment:
Thoughts on calling out some of the specs on the management port? Can we link to a page in the Gigabyte manual that covers setting up the BMC? On the Intel servers, I really want to be able to link out to the both the Lights Out Management Module and the separate BMC documentation.

Reply:
If you can figure out how to link to a page in that PDF, then sure

Reply:
@thomas-zimmerman I can link to a specific page in the PDF. In reference to setting up the BMC, would that link go on the MLAN port?

Reply:
I would think right to page 136, the start of setting up the BMC network configuration.

Reply:
I've added the links for both Eland 1U servers.